google-cloud-spanner/src/test/java/com/google/cloud/spanner/BenchmarkingDataLoaderUtility.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.spanner;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.spanner.TransactionRunner.TransactionCallable;
+import com.google.common.util.concurrent.ListenableFuture;
+import com.google.common.util.concurrent.ListeningScheduledExecutorService;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Hosts a bunch of utility methods/scripts that can be used while performing benchmarks to load
+ * data, report latency metrics, etc.
+ */
+@Category(SlowTest.class)
+@RunWith(JUnit4.class)
+public class BenchmarkingDataLoaderUtility {
+  @ClassRule public static IntegrationTestEnv env = new IntegrationTestEnv();
+  private static final String PROJECT_ID = "";
+  private static final String INSTANCE_ID = "";
+  private static final String DATABASE_ID = "";
```

Comment:
Are these maybe meant to be environment variable keys? If so, we should rename them so that is clearer (and give them an actual value, and make them actually look up an env var). Or is it something else?

Reply:
Not sure if I made it clear, but my aim here is that this script should make the lives of internal developers easier. At-least they don't have to setup a script each time. I have spoken to a few folks internally and they think such a utility makes it easy for them. Especially ones who don't have a lot of experience with Java.

So if this is an internal (for devs) utility, then we can just define the constants this way? Do you see any other advantage to setting it as env variables?